assets/src/dashboard/constants/index.js

```diff
@@ -21,6 +21,12 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import { ReactComponent as HomeWithHeart } from '../../../src/edit-story/icons/home_with_heart.svg';
+import { ReactComponent as HomeWithHeartActive } from '../../../src/edit-story/icons/home_with_heart_active.svg';
+import { ReactComponent as FloppyDisk } from '../../../src/edit-story/icons/floppy_disk.svg';
+import { ReactComponent as FloppyDiskActive } from '../../../src/edit-story/icons/floppy_disk_active.svg';
+import { ReactComponent as Compass } from '../../../src/edit-story/icons/compass.svg';
+import { ReactComponent as CompassActive } from '../../../src/edit-story/icons/compass_active.svg';
```

Comment:
this is my bad for not explaining this well, there's a section in the design system that should have these icons already. `import { ExploreTemplatesActive, ExploreTemplatesNormal ... } from '../../blah/blah/design-system'` - named 'em that way because they are specific to the dashboard.

Reply:
ah that would be way better, thanks!